Parser/Device/DeviceParserAbstract.php

```diff
@@ -82,8 +82,8 @@ abstract class DeviceParserAbstract extends ParserAbstract
         'AK' => 'Akai',
         '1A' => 'Alba',
         'AL' => 'Alcatel',
+        '20' => 'Alcor',
         '4A' => 'Aligator',
-        '3A' => 'AllDocube',
```

Comment:
Isn't AllDocube a separate brand anymore? 🤔

Reply:
Alldocube is the full name of the cube brand.

They were not divided into two companies.

I didn't find any information about splitting into two brands on the Internet

Reply:
Returned Alldocube back

Reply:
I actually didn't do a longer research for `Cube` vs `Alldocube` but if it's actually the same brand, guess it would also be fine to merge them. But we maybe shouldn't remove the brand from the list. If anyone has stored the short codes, like Matomo does, it might fail fetching the brand name for older data. So we should leave it in there (with a comment) and use it no longer...

Reply:
I think we need to additionally check the device for images

iWork 8 Ultimate
![image](https://user-images.githubusercontent.com/1337066/96157382-e9bd5d80-0f1a-11eb-8874-bd0488b314e1.png)

iWork 10 Pro
![image](https://user-images.githubusercontent.com/1337066/96157521-12455780-0f1b-11eb-8491-023e4acff97e.png)

Basically all new devices come with the Alldocube logo

Reply:
Ok. For now let's just leave them as separate brands. We can change that later as well if we want to...